components/movementsensor/movementsensor.go

```diff
@@ -154,3 +155,18 @@ func DefaultAPIReadings(ctx context.Context, g MovementSensor, extra map[string]
 
 	return readings, nil
 }
+
+// UnimplementedAccuracies returns accuracy values that will not show up on movement sensor's RC card
+// or be useable for a caller of the GetAccuracies method. The RC card currently continuosly polls accuracies,
+// so a nil error must be rturned from the GetAccuracies call.
+// It contains NaN definitiions for accuracies returned in floats, an invalid integer value for the NMEAFix of a gps
+// and an empty map of other accuracies.
+func UnimplementedAccuracies() (*Accuracy, error) {
+	return &Accuracy{
+		AccuracyMap:        map[string]float32{},
+		Hdop:               float32(math.NaN()),
+		Vdop:               float32(math.NaN()),
+		NmeaFix:            int32(-1),
```

Comment:
Looking at the bottom of the tenth page (labeled page 1-2) of [this manual](https://www.sparkfun.com/datasheets/GPS/NMEA%20Reference%20Manual-Rev2.1-Dec07.pdf), I suspect the value for an invalid fix is 0. 

I bring this up because we've got some code that assumes that all GPS-related data are values from the official documentation and doesn't check for other values correctly, and unexpected values are treated by process of elimination as the last value we handle. We can get some surprises if we start sending in unexpected data like this. Similarly, if we construct a PMTK message using this value, a parser on the other end trying to read that message might fail because `-` is not a valid start of a value.

Reply:
That is an invalid fix from a sensor that is expected. So a 0 fix is telling you your GPS is getting an invalid fix in the NMEA GGA definition, and we want that shown on the RC card. This is our way of giving a value that is not on that scale and will not show up anywhere since NMEA GGA fixes do not mean anything to non-GPS sensors.

Reply:
:shrug: leave as-is if you want, but we'll need to be careful consuming this data later.

Reply:
So we have two "invalids" going on here.
An invalid value that does not make sense in the GGA Fix nomenclature, which is returned from movement sensors that can not possibly know their fix because they are not a GPS. This is -1
An invalid value in the GGA nomenclature, which a user will want to know if their GPS is in fact returning a fix of 0 and cannot be trusted to report its position data. We want to show this invalid fix on the RC card. 

> some code that assumes that all GPS-related data are values from the official documentation and doesn't check for other values correctly

That code should only live in GPS drivers. Not in accelerometers, gyroscopes, wheeled odometry or imus.